rtloader/three/three.cpp

```diff
@@ -159,7 +164,21 @@ py_info_t *Three::getPyInfo()
     Py_XDECREF(str_path);
     return info;
 }
+/**
+ * freePyInfo()
+ */
 
+void Three::freePyInfo(py_info_t* info) {
+    if(info->version){
```

Comment:
We should do one of two things: 
1.`Two` and `Three` should be calling a C-function instead of having repeated code for `freePyInfo`.
2. This function should be implemented once in the Base class `RTLoader` and inherited.

But I don't see a strong reason why there should be code repetition.

Reply:
I did the code intentionally
1) the getPyInfo() is also repeated.  Presumably, at some point, the definitions were different
2) if the definitions diverge, then the free functions would also diverge

So, I think if we're going to combine the freePyInfo(), we should also combine the getPyInfo(), or leave them both separate & duplicated.  Thoughts?